src/macros.rs

```diff
@@ -181,6 +181,11 @@ fn return_macro_parse_failure_fallback(
         return trim_left_preserve_layout(context.snippet(span), indent, &context.config);
     }
 
+    context
+        .skipped_range
+        .borrow_mut()
+        .push((span.lo().to_usize(), span.hi().to_usize()));
```

Comment:
The is adding byte positions of the span, whereas what we really need is the line numbers. We can get a line number of the given span via `SourceMap` from libsyntax.

Reply:
Oh. I misunderstood. Thank you the review.

Reply:
Thanks for the update!